Makefile

```diff
@@ -199,6 +199,13 @@ install-deps:
 	pip install --upgrade -r requirements.txt
 	python utils/install_actor_deps.py --actor=$(ACTOR)
 
+install-deps-fedora:
+	dnf install -y git findutils python3-virtualenv
+	virtualenv --system-site-packages --python /usr/bin/$(_PYTHON_VENV) tut
```

Comment:
Regarding the installation of rpms above, in case the `_PYTHON_VENV` is not set, the default system python should be used (Python 3.7?). But not hard requirement from me.

Reply:
But to be honest, I do not know know how to do it nicely without impact on other targets. So feel free to keep it like that, just document that properly. WDYT @bocekm ?

Reply:
This concern resolved here https://github.com/oamg/leapp-repository/pull/501/commits/6f447e4484942cd0b0dcee90a11e858fc063c00b

If no PYHTON_VENV or explictly poinitng to python2, then it installs python2